openslides/motions/models.py

```diff
@@ -581,13 +581,20 @@ def set_state(self, state):
 
     def reset_state(self, workflow=None):
         """
-        Set the state to the default state.
+        Set the state to the default state. If an identifier was set
+        automatically, reset the identifier and identifier_number.
 
         'workflow' can be a workflow, an id of a workflow or None.
 
         If the motion is new and workflow is None, it chooses the default
         workflow from config.
         """
+
+        # The identifier is set automatically.
+        if config['motions_identifier'] is not 'manually':
+            self.identifier = ''
+            self.identifier_number = 0
```

Comment:
Now, the reset action removes the internal `identifier_number` AND the motion `identifier`. Manually removing of identifier in edit form is not required. Successfully tested. It's would be ok for me.
@normanjaeckel Can you live with this solution. As we discussed only the internal number should removed, right?

Reply:
This is ok for me. But `identifier_number` should be set to `None` instead of `0`.